stock_warehouse_calendar_orderpoint/models/res_company.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    orderpoint_calendar_id = fields.Many2one(
+        comodel_name="resource.calendar",
+        string="Reordering Calendar",
+        help="Calendar used to compute the lead date of reordering rules",
```

Comment:
I could do this change on `res.company` fields, not `res.config.settings` though (because of the technical purpose that the prefix `default_` has for the model's field names).
Therefore, I'd rather keep it as it is for naming consistency.

Reply:
Ok, at least the help

Reply:
Of course, already on it :wink: